blogSchema.sql

```diff
@@ -0,0 +1,16 @@
+-- create table statements
+-- A blog, with `users` and `posts`
+
+create table users (
+    id serial primary key,
+    email_address text
+);
+
+create table posts (
+    id serial primary key,
+    date timestamp,
+    subject text,
+    title text,
+    tags text,
```

Comment:
Maybe not necessary for this exercise, but consider making a separate table to hold all the tags, and another table  like this:

```sql
create table posts_tags (
  post_id integer references posts (id),
  tag_id integer references tags (id)
);
```

Then, for every tag that should be associated with a post, you just `insert` another row into `posts_tags`.

It's a more advanced technique, often referred to as a "linking table". But it can really help keep a database nice and tidy. Specifically, imagine if you needed to change the name of a tag. If you have all tags in a table, you just update that one row. If the tag is part of a each post, then you'd have to find every post that uses that tag, and update its `tags` field.